docs/user/AnsibleUsage.md

```diff
@@ -0,0 +1,71 @@
+# Using Ansible
+
+## Prereqs
+
+When using the Ansible CLI, make sure you have all the necessary tools [installed on your workstation](Dependencies.md).
+
+## Preparation
+
+### Cloud Authentication
+
+See [ansible cloud authentication](AnsibleCloudAuthentication.md) when deploying to GCP with external postgres
+
+### Variable Definitions (ansible-vars.yaml) File
+
+Prepare your `ansible-vars.yaml` file, as described in [Customize Input Values](../../README.md#customize-input-values). The only variable that cannot be set in the ansible-vars files and instead must be set with the `-e` flag is `-e CONFIG=<path_to_ansible_vars_file>`. All other values can be set either in the ansible-vars.yaml file or using the `-e` extra var param in ansible.
+
+### Initialize Ansible
+
+```bash
+# install python packages
+pip3 install --user -r requirements.txt
+
+# install ansible collections
+ansible-galaxy collection install -r requirements.yaml -f
+```
+
+## Running
+
+Declare the Actions and Task to be performed.
+
+### Actions
+
+Actions are used to install or uninstall software. One must be set when running the playbook.
+
+| Name | Description |
+| :--- | ---: |
+| Install | Installs the stack required for the specified tasks |
+| Uninstall | Uninstalls the stack required for the specified tasks |
+
+### Tasks
+
+Any number of tasks can be run at the same time. An action can run against a single task or all tasks.
+
+| Name | Description |
+| :--- | :--- |
+| baseline | Installs cluster level tooling needed for all viya deployments. These may include, cert-manager, ingress-nginx, nfs-client-provisioners and more. |
+| viya | Deploys viya |
+| cluster-logging | Installs cluster-wide logging using the [viya4-monitoring-kubernetes](https://github.com/sassoftware/viya4-monitoring-kubernetes) project. |
+| cluster-monitoring | Installs cluster-wide monitoring using the [viya4-monitoring-kubernetes](https://github.com/sassoftware/viya4-monitoring-kubernetes) project. |
+| viya-monitoring | Installs viya namespace level monitoring using the [viya4-monitoring-kubernetes](https://github.com/sassoftware/viya4-monitoring-kubernetes) project. |
+
+### Examples
```

Comment:
There are 4 examples in the DockerUsage.md file. I think it would be valuable to cover all of the same scenarios in this file too.

Reply:
updated